README.md

```diff
@@ -79,6 +84,9 @@ To enable logging on the client-side, just:
 
     (setq lsp-log-io t)
 
+Remember that the Erlang Language Server requires Erlang/OTP 21 or
```

Comment:
This is also in the VS Code section but it's easy to miss and very important, so it's worth repeating here.

Reply:
Created #275 to expand a bit on this.